src/opnsense/mvc/app/models/OPNsense/Unbound/Unbound.xml

```diff
@@ -54,13 +54,18 @@
                 <Multiple>Y</Multiple>
             </outgoing_interface>
             <enable_wpad type="BooleanField"/>
+            <builtin_hints type="BooleanField"/>
         </general>
         <advanced>
             <hideidentity type="BooleanField"/>
             <hideversion type="BooleanField"/>
             <prefetch type="BooleanField"/>
             <prefetchkey type="BooleanField"/>
             <dnssecstripped type="BooleanField"/>
+            <aggressivensec type="BooleanField">
+                <Required>Y</Required>
```

Comment:
this requires a model version bump

Reply:
oops. sure, I'll fix it now. so should I remove the hints part?

Reply:
for now yes but we should add it if the problem continues and upstream is unresponsive

Reply:
done!)